### PR TITLE
update user object

### PR DESCRIPTION
**Pull request by tracytrle**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ help:
 	@echo "  install - Install dependencies"
 	@echo "  run     - Run the Flask application"
 	@echo "  clean   - Remove virtual environment"
-	@echo "  dbmigrate - Run the Flask migration"
+	@echo "  dbupgrade - Run the Flask upgrade"
```
Comment (travistrle) [inline Makefile:18]: Can you add one row for db migration here?

File: Makefile [hunk 2]
```diff
@@ -40,12 +40,19 @@ clean:
 	@echo "Removing virtual environment..."
 	rm -rf $(VENV_DIR)
 
+
 dbmigrate:
+	@echo "Running the Flask migration with message: $(m)"
```
Comment (travistrle) [inline Makefile:45]: nice

File: medicai/auth/routes.py [hunk 3]
```diff
@@ -16,16 +16,19 @@ def index():
 @cross_origin()
 def register():
     data = request.get_json()
-    user_exists = User.query.filter_by(email=data['email']).first() is not None
+    user_exists = User.query.filter_by(phone=data['phone']).first() is not None
```
Comment (travistrle) [inline medicai/auth/routes.py:19]: Did we index this column?

File: medicai/user/routes.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+from flask import request, jsonify
+from flask_jwt_extended import create_access_token, jwt_required, get_jwt_identity
+from flask_cors import CORS, cross_origin
+from medicai.models.user import User
+from medicai.extensions import db
+from medicai.extensions import bcrypt
+from medicai.auth import bp
+
+# @db.route('/user/register', methods=['POST'])
```
Comment (travistrle) [inline medicai/user/routes.py:9]: Can you remove this?

**Review by travistrle — COMMENTED** [review]
We can index the phone column in the next PR. But highly recommend to do it since we're gonna query a lot on that column.
You may also want to add a unique constraint on the phone since no two users should use same phone number.

File: medicai/auth/routes.py (hunk 3 above)
Comment (tracytrle) [reply medicai/auth/routes.py:19]: later

File: medicai/user/routes.py (hunk 4 above)
Comment (tracytrle) [reply medicai/user/routes.py:9]: done

File: Makefile (hunk 2 above)
Comment (tracytrle) [reply Makefile:45]: later

**Pull request merged**